releasenotes/notes/otel-go-runtime-metrics-dd977054dd009f99.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    The OTLP metrics intake now maps OpenTelemetry Go runtime metrics to their Datadog counterparts.
```

```suggestion
    The OTLP metrics converter now maps OpenTelemetry Go runtime metrics to their Datadog counterparts.
```
"intake" might be confused with the OTLP Ingest, or something else 😉 This however applies to both the exporter and the ingest, right?